action.yml

```
@@ -234,6 +239,22 @@ runs:
           fi
         fi
 
+    - if: ${{ inputs.cache == 'true' }}
+      id: gen-cache-key
+      shell: python
+      run: |
+        import hashlib, os, pathlib
+        fp = pathlib.Path(os.path.join(os.environ["GITHUB_ACTION_PATH"], 'requirements.txt'))
+        with open(os.environ["GITHUB_OUTPUT"], "a") as f:
+          print('hash=%s' % (hashlib.sha1(fp.read_bytes()).hexdigest()), file=f)
```

needs a LF for each key-value pair
```suggestion
          print('hash=%s\n' % (hashlib.sha1(fp.read_bytes()).hexdigest()), file=f)
```